### PR TITLE
Feature/strain method for symbol

### PR DESCRIPTION
**Pull request by timheilman**
Let me know what you think :)

### CODE REVIEW

**Comment by danielevans** [comment]
Love it. This would be the perfect situation for a coercion method, I think. I'll have to take a crack at them. 

One comment is the name, which might is a bit odd since it is `for_symbol` but takes a string.

**Comment by timheilman** [comment]
eh? It takes a symbol...

I looked up coercion methods, but am not seeing how they'd apply here. All the examples I can find involve operator methods, which Strain is missing.

**Comment by danielevans** [comment]
I misread the code :) my bad.

A Coercion methods would allow this: `Bridge::Strain(:S)` to give Spade

File: lib/bridge/strain.rb [hunk 1]
```diff
@@ -36,6 +36,21 @@ def self.minors
       [Club, Diamond]
     end
 
+    @mapping = {}
+    %w(Clubs Diamonds Hearts Spades NoTrumps
```
Comment (danielevans) [inline lib/bridge/strain.rb:40]: See? Aren't aligned columns so.... satisfying?

**Review by danielevans — APPROVED** [review]

File: lib/bridge/strain.rb (hunk 1 above)
Comment (timheilman) [reply lib/bridge/strain.rb:40]: Ha!  I was trying to demonstrate I can, actually, contribute while conforming to the surrounding code's existing style.

One book-learnin' thing I've been gnawing on was this one from Robert C. Martin: if I find myself fiddling with horizontal alignment, the problem is the length of the lists, not the lack of alignment.

Comment (danielevans) [reply lib/bridge/strain.rb:40]: I find horizontal alignment pleasing if there are multiple lines with similar structure. And my editor has a built-in horizontal alignment utility, so I'm rarely doing it manually (except when adding a line to an already-aligned block).

However, I *do* lean toward meta-programming or similar tactics to compact those lists into 1 line when possible. 

Here is some code from my current task:
```
  let(:serial_number) { SecureRandom.uuid }
  let(:store) { create(:store) }
  let(:position) { create(:position, store: store) }
  let(:security_device) { create(:security_device, store: store, position: position) }
  let(:status_id) { SecureRandom.uuid }
```
`C-u M-x align-regexp` and:

```
  let(:serial_number)   { SecureRandom.uuid        }
  let(:store)           { create(:store)           }
  let(:position)        { create(:position,        store: store) }
  let(:security_device) { create(:security_device, store: store, position: position) }
  let(:status_id)       { SecureRandom.uuid        }
```

Comment (danielevans) [reply lib/bridge/strain.rb:40]: @timheilman which book of Robert C. Martin's are you reading? Based on the tidbits you've brought up I suspect I'd learn a lot reading it.

Comment (timheilman) [reply lib/bridge/strain.rb:40]: @danielevans Clean Code.
I've just started The Pragmatic Programmer as well.
Did a lot of studying of Ruby today as well, trying to comprehend method lookup and the inheritance chain, and a few talks from Jim Weirich.

**Pull request merged**